packages/domain/lib/src/exceptions/exceptions.dart

```diff
@@ -0,0 +1,69 @@
+
+
+class AppException implements Exception {
+  final String code;
+  final String message;
+  final String? details;
+
+  AppException(
+      {required this.code, required this.message, required this.details});
```

Comment:
comma

Reply:
if expression sized in one string, I think comma is no needed. but okay) corrected